### PR TITLE
GH-4321 check spdx in header

### PR DESCRIPTION
**Pull request by JervenBolleman**
GitHub issue resolved: #4321 
Briefly describe the changes proposed in this PR:

Checks that the Copyright header is present and has the SPDX line

----
PR Author Checklist (see the [contributor guidelines](https://github.com/eclipse/rdf4j/blob/main/CONTRIBUTING.md) for more details):

 - [x] my pull request is [self-contained](https://rdf4j.org/documentation/developer/merge-strategy/#self-contained-changes-pull-requests-and-commits)
 - [x] I've added tests for the changes I made
 - [x] I've applied [code formatting](https://github.com/eclipse/rdf4j/blob/main/CONTRIBUTING.md#code-formatting) (you can use `mvn process-resources` to format from the command line)
 - [x] I've [squashed](https://rdf4j.org/documentation/developer/squashing) my commits where necessary 
 - [x] every commit message starts with the issue number (GH-xxxx) followed by a meaningful description of the change

### CODE REVIEW

File: testsuites/lucene/src/main/java/org/eclipse/testsuite/rdf4j/sail/lucene/AbstractLuceneSailTest.java [hunk 1]
```diff
@@ -1,11 +1,13 @@
-/**
- * Copyright (c) 2016 Eclipse RDF4J contributors.
+/*******************************************************************************
+ * Copyright (c) 2019 Eclipse RDF4J contributors.
```
Comment (hmottestad) [inline testsuites/lucene/src/main/java/org/eclipse/testsuite/rdf4j/sail/lucene/AbstractLuceneSailTest.java:2]: Wrong year?

File: core/sparqlbuilder/src/test/java/org/eclipse/rdf4j/sparqlbuilder/constraint/ExtendedExpressionsTest.java [hunk 2]
```diff
@@ -1,13 +1,13 @@
-/*
- * ******************************************************************************
- *  * Copyright (c) 2021 Eclipse RDF4J contributors.
- *  *
+/*******************************************************************************
+ * Copyright (c) 2019 Eclipse RDF4J contributors.
+ *
```
Comment (hmottestad) [inline core/sparqlbuilder/src/test/java/org/eclipse/rdf4j/sparqlbuilder/constraint/ExtendedExpressionsTest.java:3]: Wrong year?

File: core/sparqlbuilder/src/main/java/org/eclipse/rdf4j/sparqlbuilder/constraint/propertypath/builder/PropertyPaths.java [hunk 3]
```diff
@@ -1,13 +1,13 @@
-/*
- * *****************************************************************************
- * Copyright (c) 2015 Eclipse RDF4J contributors, Aduna, and others.
+/*******************************************************************************
+ * Copyright (c) 2021 Eclipse RDF4J contributors.
```
Comment (hmottestad) [inline core/sparqlbuilder/src/main/java/org/eclipse/rdf4j/sparqlbuilder/constraint/propertypath/builder/PropertyPaths.java:2]: I think this should probably still have the old "Aduna, and others." bit. Also I think you changed the year as well.

**Review by hmottestad — CHANGES_REQUESTED** [review]
Great that you are fixing this!

I saw that you changed the copyright year on a couple of files and also removed the "Aduna, and others" text form at least one file. I presume this is just copy-paste issues. I didn't look through all the files though, so there might be more than what I found.

Also noticed that this is targeted at the develop branch. Was that intentional?

File: core/sparqlbuilder/src/main/java/org/eclipse/rdf4j/sparqlbuilder/constraint/propertypath/builder/PropertyPaths.java (hunk 3 above)
Comment (JervenBolleman) [reply core/sparqlbuilder/src/main/java/org/eclipse/rdf4j/sparqlbuilder/constraint/propertypath/builder/PropertyPaths.java]: fixed

File: core/sparqlbuilder/src/test/java/org/eclipse/rdf4j/sparqlbuilder/constraint/ExtendedExpressionsTest.java (hunk 2 above)
Comment (JervenBolleman) [reply core/sparqlbuilder/src/test/java/org/eclipse/rdf4j/sparqlbuilder/constraint/ExtendedExpressionsTest.java]: fixed

File: testsuites/lucene/src/main/java/org/eclipse/testsuite/rdf4j/sail/lucene/AbstractLuceneSailTest.java (hunk 1 above)
Comment (JervenBolleman) [reply testsuites/lucene/src/main/java/org/eclipse/testsuite/rdf4j/sail/lucene/AbstractLuceneSailTest.java]: fixed

**Comment by JervenBolleman** [comment]
Fixed and double checked the copyright year, and aduna accidental changes.

I targeted develop to avoid the risk of differences in develop/main breaking unrelated open PR.

**Review by jeenbroekstra — APPROVED** [review]
Nice!

**Comment by JervenBolleman** [comment]
@hmottestad would you mind having a look to see if I resolved your concern?

**Pull request merged**